scripts/src/blockchain/errors.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
```

Comment:
same, this one was checked in by mistake (probably in tests there is an import of .ts)